crates/e2e-testing/src/asserts.rs

```diff
@@ -0,0 +1,44 @@
+use anyhow::Result;
+
+pub fn assert_status(url: &str, expected: u16) -> Result<()> {
+    let resp = req(url)?;
+    let status = resp.status();
+    let body = resp.text()?;
+    assert_eq!(status, expected, "{}", body);
+    Ok(())
+}
+
+pub fn assert_http_request(
```

Comment:
would it be more appropriate to call this `assert_http_response` since we are asserting against the response?

Reply:
sure, renamed.